tool/planet/main.go

```diff
@@ -201,6 +201,16 @@ func run() error {
 		cstatusClientKeyFile = cstatus.Flag("client-key-file", "mTLS client key file").
 					Default(ClientRPCKeyPath).OverrideDefaultFromEnvar(EnvPlanetAgentClientKeyFile).String()
 
+		cstatusDump        = app.Command("debug-status", "Dump Planet Agent debug internals")
+		cstatusDumpProfile = cstatusDump.Flag("profile", "Name of the profile to dump").Required().String()
```

Comment:
Providing invalid profile arg just outputs```[ERROR]: invalid profile: ...```
Maybe we could list valid profiles or add link to 
https://golang.org/pkg/runtime/pprof/#Profile

Reply:
It is not really meant to be user-facing. And I'll have `goroutine` be a default. Still unsure we need the rest of the profiles.